test/wpackettest.c

```diff
@@ -590,7 +590,7 @@ static int test_WPACKET_quic_vlint_random(void)
         if (!TEST_int_gt(RAND_bytes(rand_data, sizeof(rand_data)), 0))
             return cleanup(&pkt);
 
-        memcpy(&expected, rand_data, sizeof(uint64_t));
+        memcpy(&expected, rand_data, sizeof(expected));
```

Comment:
This seems to be an unrelated change - i.e. unrelated to the rest of this PR ... I'd tend to leave it out.
It's harmless - but I also prefer sizeof(type) rather than sizeof(var) ...

Reply:
We generally use sizeof(var) as a rule.  From the style guide:

> For sizeof, use a variable when at all possible

Reply:
Yes, this is generally safer - if the type is changed the `sizeof()` is still correct